helper/parseItems.js

```diff
@@ -83,6 +83,27 @@ const postParseItems = async (profileData, items) => {
       items.pets.push(newPet);
     }
   }
+
+  // Parse Museum
+  items.museum = [];
+  if (museumData?.items) {
+    for (const data of Object.values(museumData.items)) {
+      if (data.borrowing) continue;
+      if (data.items?.data === undefined) continue;
+
+      const decodedItem = await decodeData(data.items.data);
+
+      items.museum.push(...decodedItem);
+    }
+
+    for (const data of museumData.special) {
```

Comment:
`museumData.special` can not exist

Fix:
`museumData.special` -> `museumData.special || []`

Reply:
fixed